Update Xero app partnerships docs (#264)
codatio/codat-docs: docs/integrations/accounting/xero/xero-app-partner-program.md

---
title: "Joining the Xero App Partner Program"
description: "Unlock the full value of connectivity with Xero by becoming a Xero App Partner"
---

import LocalStorageChecklist from "@components/global/LocalStorageChecklist";
import BrowserOnly from "@docusaurus/BrowserOnly";

If you want to have more than 25 Xero connections, you'll need to join the **Xero App Partner Program**.

Becoming a certified Xero App Partner brings the following benefits:

- No limit to the number of companies (SMBs) you can connect to via your Xero integration.
- The opportunity to reach millions of subscribers and advisors through the [Xero App Store](http://www.xero.com/marketplace) and [app partner program](https://developer.xero.com/app-partner).

To join the program, there are a series of requirements your integration must first satisfy. There are also some additional use case-specific requirements, as explained in [Xero App Partner guidance by use case](/integrations/accounting/xero/accounting-xero-app-partner-guidance).

> We've broken down the requirements you need to action based on the [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/#required-for-all-integrations). During your App Partner Program application, you may use the answers on this page to facilitate your application. [We take care of a number of checkpoints for you](/integrations/accounting/xero/xero-app-partner-program#we-take-care-of).

Reviewer: ```suggestion
We've broken down the requirements you need to action based on the [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/#required-for-all-integrations). You can use the answers on this page to help with your App Partner Program application. [We take care of a number of checkpoints for you](/integrations/accounting/xero/xero-app-partner-program#we-take-care-of).
```

:::caution Use cases

There are additional checkpoints based on use cases.

:::

<details>
<summary><b>Additional resources</b></summary>

In the Xero developer documentation, see [Steps to becoming an app partner](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/app-partner-steps) and the [App Partner Program FAQs](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/faqs/).

> For tips and advice, read our blog post on [How to get your app certified on the Xero App Store](https://www.codat.io/blog/get-your-app-certified-on-the-xero-app-store/).

Reviewer: We should probably just look to ensure all the relevant content is in the docs (not now)
